ERCS/erc-7739.md

```diff
@@ -268,21 +271,11 @@ hash =
 
 The `PersonalSign` workflow does not require additional data to be appended to the `signature` passed into `isValidSignature`.
 
-### `supportsNestedTypedDataSign` function for detection
+### Support detection
 
-To facilitate automatic detection, smart accounts SHOULD implement the following function:
+Smart accounts SHOULD return `bytes4(0x77390001)` for `isValidSignature(0x7739773977397739773977397739773977397739773977397739773977397739, "Any Signature")` to indicate support for this standard.
```

Comment:
I'd consider requiring the signature argument to be the empty string instead of accepting anything. Just to reduce the surface where `isValidSignature` behaves different than expected.

```suggestion
Smart accounts SHOULD return `bytes4(0x77390001)` for `isValidSignature(0x7739773977397739773977397739773977397739773977397739773977397739, "")` to indicate support for this standard.
```